common/djangoapps/student/forms.py

```diff
@@ -115,12 +115,11 @@ class AccountCreationForm(forms.Form):
     validation, not rendering.
     """
     # TODO: Resolve repetition
-    username = forms.SlugField(
+    username = forms.CharField(
```

Comment:
@griffresch -- would it be okay to keep the Public Username field limited to A-Z, a-z, and 0-9 characters, while ensuring all of the other fields accept full unicode?